src/struct/commands/CommandUtil.js

```diff
@@ -115,49 +112,47 @@ class CommandUtil {
 
     /**
      * Sends a response, overwriting the last response.
-     * @param {string} [content=''] - Content to send.
-     * @param {MessageOptions|MessageAdditions} [options={}] - Options to use.
+     * @param {string|MessageOptions} [options={}] - Options to use.
      * @returns {Promise<Message|Message[]>}
      */
-    async sendNew(content, options) {
-        const sent = await this.message.channel.send(this.constructor.transformOptions(content, options));
+    async sendNew(options) {
+        const sent = await this.message.channel.send(this.constructor.transformOptions(options));
         const lastSent = this.setLastResponse(sent);
         this.setEditable(!lastSent.attachments.size);
         return sent;
     }
 
     /**
-     * Sends a response with a mention concantenated to it.
-     * @param {string} [content=''] - Content to send.
-     * @param {MessageOptions|MessageAdditions} [options={}] - Options to use.
+     * Sends a response replying to the user's message.
+     * @param {string|MessageOptions} [options={}] - Options to use.
      * @returns {Promise<Message|Message[]>}
      */
-    reply(content, options) {
-        return this.send(this.constructor.transformOptions(content, options, { reply: this.message.member || this.message.author }));
+    reply(options) {
+        return this.send({
+            reply: { messageReference: this.message, failIfNotExists: false },
+            ...this.constructor.transformOptions(options)
+        });
     }
 
     /**
      * Edits the last response.
-     * @param {string} [content=''] - Content to send.
-     * @param {MessageEditOptions|MessageEmbed} [options={}] - Options to use.
+     * @param {string|MessageEditOptions} [options={}] - Options to use.
      * @returns {Promise<Message>}
      */
-    edit(content, options) {
-        return this.lastResponse.edit(content, options);
+    edit(options) {
+        return this.lastResponse.edit(options);
     }
 
     /**
      * Transform options for sending.
-     * @param {string} [content=''] - Content to send.
-     * @param {MessageOptions|MessageAdditions} [options={}] - Options to use.
-     * @param {MessageOptions} [extra={}] - Extra options to add.
+     * @param {string|MessageOptions} [options={}] - Options to use.
      * @returns {MessageOptions}
      */
-    static transformOptions(content, options, extra) {
-        const transformedOptions = APIMessage.transformOptions(content, options, extra);
-        if (!transformedOptions.content) transformedOptions.content = null;
-        if (!transformedOptions.embed) transformedOptions.embed = null;
-        return transformedOptions;
+    static transformOptions(options) {
```

Comment:
Shallow clone the options before mutating it.

Reply:
Also my bad.